Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeDrawingAreaProxyIOS.mm

```diff
@@ -41,6 +41,7 @@
 @interface WKDisplayLinkHandler : NSObject {
     WebKit::RemoteLayerTreeDrawingAreaProxy* _drawingAreaProxy;
     CADisplayLink *_displayLink;
+    CFTimeInterval _maximumRefreshRate;
```

Comment:
A "rate" should not be a time interval. A "rate" is a frames-per-second value that we normally store as an integer. If this is a frame duration, let's give it a good name in our code (and file a bug about the bad SPI).

Reply:
Or just add a comment so future me is not confused.